troubleshoot/index.md

```diff
@@ -57,14 +57,48 @@ If you have an [Elastic subscription](https://www.elastic.co/pricing), you can c
   If you contact us by email, use the email address you registered with so we can help you more quickly. If your registered email is a distribution list, you can register a second email address with us. Just open a case to let us know the name and email address you want to add.
   :::
 
+  :::{warning}
+  All cases opened by email default to normal severity. For incidents, open a case through the [Elastic Support Portal](https://support.elastic.co/) and select the [appropriate severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
+  :::
+
 ## Working with support [troubleshoot-work-with-support]
 
-Try these tips when opening a support case:
+When you open a support case:
+
+* Describe the problem and its [contextual severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
```

Comment:
```suggestion
* Describe the problem with context about the situation, severity, and seriousness, as described in [What should I say in my case?](https://www.elastic.co/support/welcome#what-to-say-in-a-case).
```
"Contextual severity" isn't super clear to me so how about the above, to better match the article we're linking to?

Reply:
Ship it! ⛵